Fix panic on armhf/v7 due to bug in zstd causing address-space exhausition.

https://codeberg.org/forgejo/forgejo/issues/8165
diff --git a/build/generate-bindata.go b/build/generate-bindata.go
index 67d3776..e9b7bfd 100644
--- a/build/generate-bindata.go
+++ b/build/generate-bindata.go
@@ -124,7 +124,7 @@ type direntry struct {
 }
 
 func generate(fsRoot fs.FS, packageName string, globalTime bool, output io.Writer) error {
-	enc, err := zstd.NewWriter(nil, zstd.WithEncoderLevel(zstd.SpeedBestCompression))
+	enc, err := zstd.NewWriter(nil, zstd.WithLowerEncoderMem(true))
 	if err != nil {
 		return err
 	}
